head/head.go

```diff
@@ -86,6 +88,8 @@ func NewHead(ctx context.Context, options ...opts.Option) (*Head, chan Bootstrap
 		libp2p.DefaultMuxers,
 		libp2p.Transport(quic.NewTransport),
 		libp2p.Transport(tcp.NewTCPTransport),
+		libp2p.Security(tls.ID, tls.New),
+		libp2p.Security(noise.ID, noise.New),
```

Comment:
go-libp2p prefers Noise over TLS (for better performance with non-go libp2p implementations), however go-ipfs prefers TLS over Noise and is the predominant IPFS implementation that utilizes the DHT so IMO we should prefer TLS.